Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -71,7 +71,15 @@ void EventRegionContext::unite(const FloatRoundedRect& roundedRect, RenderObject
     auto layerBounds = transformAndClipIfNeeded(rect, [](auto affineTransform, auto rect) {
         return affineTransform.mapRect(rect);
     });
-    uniteInteractionRegions(renderer, layerBounds);
+
+    std::optional<FloatRect> clipRect;
+    if (!m_clipStack.isEmpty())
+        clipRect = m_clipStack.last();
+    std::optional<AffineTransform> transform;
+    if (!m_transformStack.isEmpty())
+        transform = m_transformStack.last();
```

Comment:
Is just grabbing the last transform on the stack OK? What transform do you need; a root-relative one, or one relative to some ancestor? Which ancestor?

Reply:
We want to apply the same transform / clipping as the one applied to the layer bounds in `transformAndClipIfNeeded`. (it only uses the last clip rect / transform)

I'm adding a couple of tests with nested transforms